src/components/Inbox/VoiceCall/StartVoiceCallPopover.tsx

```diff
@@ -0,0 +1,59 @@
+import { useQuery } from "@apollo/client";
+import { Button, Popover } from "antd";
+import { GetVoiceCallTimesDocument } from "src/graphql/queries/GetVoiceCallTimes.generated";
+import { millisecondsToMinutesAndSeconds, reportApolloError } from "src/utils";
+
+interface Props {
+  onClose: () => void;
+  onConfirm: () => void;
+  isOpen: boolean;
+}
+
+const StartVoiceCallPopover: React.FC<Props> = ({
+  onConfirm,
+  onClose,
+  isOpen,
+  children,
+}) => {
+  const { data } = useQuery(GetVoiceCallTimesDocument, {
+    onError: (err) =>
+      reportApolloError(err, "StartVoiceCallPopover/GetVoiceCallTimesDocument"),
```

Comment:
slight improvement from earlier today:

```suggestion
    onError: errorReporter("StartVoiceCallPopover/GetVoiceCallTimesDocument"),
```

Reply:
I see that's in DOC client, not Pathways yet.

Reply:
yep my bad!